R/anovarepeatedmeasures.R

```diff
@@ -437,16 +437,19 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
 
   withinIndices <- .mapAnovaTermsToTerms(rownames(withinAnovaTable), rownames(corrections))
 
-  ggTable[["num Df"]]          <- withinAnovaTable[["num Df"]] * corrections[withinIndices, "GG eps"]
+  ggCorrections <- pmin(corrections[withinIndices, "GG eps"], 1)
+  hfCorrections <- pmin(corrections[withinIndices, "HF eps"], 1)
```

Comment:
While I get what you're doing now, I probably wouldn't understand why anymore in a month from now. So perhaps this deserves a comment along the lines of 
```suggestion
  # set 1 as an upper-bound of the correction factors, see https://github.com/jasp-stats/jasp-issues/issues/1709
  ggCorrections <- pmin(corrections[withinIndices, "GG eps"], 1)
  hfCorrections <- pmin(corrections[withinIndices, "HF eps"], 1)
```